### PR TITLE
Create the example for drawing and moving

### PR DESCRIPTION
**Pull request by chrehall68**
This creates the full example for Ch. 2.3 in the pygame course.

### CODE REVIEW

File: games/chapter2/examples/draw_objects.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        if event.type == pygame.QUIT:
+            run = False
+
+    # move a rectangle that isn't a pygame.Rect object
+    x += 1  # move to the right 1 px
+    y += 1  # move down 1 px
+
+    # move a rectangle that is a pygame.Rect object
+    green_rectangle.move_ip(1, 2)  # moves 1 to the right, 2 down
+    # this is equivalent to green_rectangle = green_rectangle.move(1, 2)
```
Comment (hpanchal092) [inline games/chapter2/examples/draw_objects.py:36]: i ran a quick test, `move_ip()` is not the same as `move()`. the `move()` method moves the rectangle to the specific coordinate, and the `move_ip()` will move the rectangle relative to where it already is. we should change this comment as well as edit the notion to fix that

**Review by hpanchal092 — CHANGES_REQUESTED** [review]

File: games/chapter2/examples/draw_objects.py (hunk 1 above)
Comment (hpanchal092) [reply games/chapter2/examples/draw_objects.py:36]: everything else lgtm 👍

Comment (hpanchal092) [reply games/chapter2/examples/draw_objects.py:36]: nvm im a huge dumbass just realized the second example is equating to itself

**Review by hpanchal092 — APPROVED** [review]
LGTM!

**Review by Aadya-Jain — APPROVED** [review]
LGTM!

**Pull request merged**